### DISCUSSION TITLE
"Show High Alchemy Profit" should account for GE tax

### DISCUSSION THREAD

**Discussion by fuzxi** [runelite/runelite#15002] [Ideas]
**Is your feature request related to a problem? Please describe.**
The 'Show High Alchemy Profit" option, in the Item Prices plugin, does not account for the 1% GE tax on items valued over 100gp when determining the potential profit from alching an item. For instance, magic longbows currently have a GE value of 1344 and high alch for 1536. As nature runes currently have a GE value of 200, the plugin displays the high alch profit as -8. However, due to the GE tax, selling a magic longbow for 1344 would only net 1331 coins, making the high alch profit +5. This can make a big difference when choosing whether to alch or sell large stacks of items, especially with valuable items that hover around alch value, such as many dragon items.

**Describe the solution you'd like**
When calculating high alch profit, the GE value of the item should be multiplied by 0.99 if its initial value is greater than or equal to 100.

**Describe alternatives you've considered**
Manually checking and comparing high alch values for items worth less than 100gp. That defeats the point of having the plugin option, though.

**Additional context**
N/A

**Comment by Nightfirecat** [comment]
What you're proposing is not the original intent of the feature. It currently works as a way to determine if an item is worth buying from the GE and alching with nature runes also bought from the GE.

**Comment by fuzxi** [comment]
I see. I've always considered it more geared towards selling than buying, because the information is only shown when you already have one in your bank/inventory and not in the GE offer screen. 
Would it be feasible to add an option to toggle accounting for the tax?
